src/motor-interface-constants.h

```diff
@@ -95,6 +95,12 @@
 #define PIN_P8_19  (GPIO_0_BASE | 22)  // EHRPWM2A
 #define PIN_P8_26  (GPIO_1_BASE | 29)
 
+#define PIN_P8_43  (GPIO_2_BASE | 8) //Check http://www.ofitselfso.com/BeagleNotes/pinmux.pdf, select the GPIO device and the number
+#define PIN_P8_44  (GPIO_2_BASE | 9)
+#define PIN_P8_40  (GPIO_2_BASE | 13)
+#define PIN_P8_39  (GPIO_2_BASE | 12)
+#define PIN_P8_28  (GPIO_2_BASE | 24)
```

Comment:
Please sort these by pin number like the other PIN_P8_* defines.
Also please add a comment about the "standard" pin name like the others (these are all LCD_* pins).